content/projects/understanding-loops/_index.md

```diff
@@ -13,16 +13,50 @@ tags:
 title: Shopping cart calculations
 ---
 
-## Directory structure
+## Language specific notes
 
 ### JavaScript
 
 Your directory structure should look like this:
 
 ```
-├── src
-    ├── shopping_cart.js
-    └── data.json
+└── src
```

Comment:
In the previous version of this project, we told the learners where to put the data. I don't think it's necessary to be specific there because it doesn't matter to the automarker and it is really just a matter of style.

Reply:
Agreed, they can put it anywhere really

Reply:
Since we are specific on the javaScript part lets say they can also put it under src -> java -> data.json where all the other files are, what do you think?